turbopack/crates/turbopack-css/src/chunk/mod.rs

```diff
@@ -78,7 +78,25 @@ impl CssChunk {
             writeln!(body, "/* {} */", id)?;
             let close = write_import_context(&mut body, content.import_context).await?;
 
-            body.push_source(&content.inner_code, content.source_map.map(Vc::upcast));
+            let source_map = if *self
+                .chunking_context()
+                .should_use_file_source_map_uris()
+                .await?
+            {
+                let source_map = content.source_map.map(|m| m.generate_source_map());
+                match source_map {
+                    Some(map) => {
+                        (*(normalize_source_map(map, self.chunking_context().context_path())
+                            .await?))
+                            .map(Vc::upcast)
+                    }
+                    None => None,
+                }
+            } else {
+                content.source_map.map(Vc::upcast)
+            };
```

Comment:
It would be more efficient when switching that: First check the Option, afterwards the file uris:

```
let source_map = if let Some(source_map) = source_map {
  if *self
    .chunking_context()
    .should_use_file_source_map_uris()
    .await?
    ...
} else {
  None
}
```